jena-text/testing/TextQuery/text-config.ttl

```diff
@@ -31,6 +31,7 @@ text:TextIndexLucene  rdfs:subClassOf   text:TextIndex .
 
 <#indexLucene> a text:TextIndexLucene ;
     text:directory "mem" ;
+    text:multilingualSupport true ;
```

Comment:
Well, I was testing how multilingual works and since making this change did not break anything, I left it there. I can revert it back.